src/Repository/ArticleRepository.php

```diff
@@ -37,33 +37,21 @@ public function remove(Article $entity, bool $flush = false): void
         if ($flush) {
             $this->getEntityManager()->flush();
         }
-
+    }
+    public function save(Article $article)
+    {
+        $this->getEntityManager()->persist($article);
+        $this->getEntityManager()->flush();
+    }
+    public function delete(Article $content)
+    {
+        $this->getEntityManager()->remove($content);
+            $this->getEntityManager()->flush();
     }
 
-
-//    /**
-//     * @return Article[] Returns an array of Article objects
-//     */
-//    public function findByExampleField($value): array
-//    {
-//        return $this->createQueryBuilder('a')
-//            ->andWhere('a.exampleField = :val')
-//            ->setParameter('val', $value)
-//            ->orderBy('a.id', 'ASC')
-//            ->setMaxResults(10)
-//            ->getQuery()
-//            ->getResult()
-//        ;
-//    }
-
-//    public function findOneBySomeField($value): ?Article
-//    {
-//        return $this->createQueryBuilder('a')
-//            ->andWhere('a.exampleField = :val')
-//            ->setParameter('val', $value)
-//            ->getQuery()
-//            ->getOneOrNullResult()
-//        ;
-//    }
-
+    public function update(Article $content)
+    {
+        $this->getEntityManager()->persist($content);
+        $this->getEntityManager()->flush();
```

Comment:
->flush($content)

flush bez parametrow powoduje zapis do bazy wszystkich obiektow, ktore sa w Entitymanager, jest to malo bezpieczne i niewydajne.